src/tests/routes/comments.test.js

```diff
@@ -0,0 +1,216 @@
+/* eslint-disable no-undef */
+/* eslint-disable import/no-extraneous-dependencies */
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import * as Factory from '../../helpers/factory';
+import status from '../../config/status';
+import db from '../../models';
+import app from '../../app';
+
+chai.use(chaiHttp);
+chai.should();
+
+let createdUser = {};
+let createdArticle = {};
+let createdComment = {};
+let commentId;
+
+const newUser = Factory.user.build();
+const newArticle = Factory.article.build();
+const newComment = Factory.comment.build();
+
+delete newUser.id;
+delete newArticle.id;
+delete newComment.id;
+
+describe('COMMENTS', () => {
+  before(async () => {
+    try {
+      await db.User.destroy({
+        truncate: true,
+        cascade: true,
+        logging: false
+      });
+      await db.Article.destroy({
+        truncate: true,
+        cascade: true,
+        logging: false
+      });
+      await db.Comment.destroy({
+        truncate: true,
+        cascade: true,
+        logging: false
+      });
+      createdUser = (await db.User.create(newUser, { logging: false })).dataValues;
+      newArticle.userId = createdUser.id;
+      createdArticle = (await db.Article.create(newArticle, { logging: false })).dataValues;
+      newComment.articleId = newArticle.id;
+      // createdComment = (await db.Comment.create(newComment, { logging: false })).dataValues;
```

Comment:
@MBenedicte delete the comment if not in use

Reply:
Yes.